### PR TITLE
hub: add open fleets metadata

### PR DESCRIPTION
**Pull request by tmigone**
This PR introduces changes needed for openFleets.

Change-type: patch
Signed-off-by: Tomás Migone <tomas@balena.io>

### CODE REVIEW

**Review by chrisys — APPROVED** [review]

**Comment by tmigone** [comment]
@balena-ci retest

**Comment by tmigone** [comment]
@balena-ci retest

**Comment by tmigone** [comment]
@balena-ci retest

**Comment by tmigone** [comment]
@balena-ci retest

File: balena.yml [hunk 1]
```diff
@@ -3,6 +3,13 @@ type: sw.application
 description: >-
   Build a single or multi-room streamer for an existing audio device using a
   Raspberry Pi! Supports Bluetooth, Airplay and Spotify Connect
+fleetcta: Sounds good
+post-provisioning: >-
+  # Usage instructions
```
Comment (iamsolankiamit) [inline balena.yml:8]: lets make these as h2(`##`)

Comment (tmigone) [reply balena.yml:8]: oh good catch

**Pull request merged**